_infra/helm/ras-rm-benchmark/templates/job.yaml

```diff
@@ -19,8 +19,10 @@ spec:
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           env:
             - name: GOOGLE_CLOUD_PROJECT
-              value: "ras-rm-performance-tests"
+              value: "ras-rm-performance-20220908"
```

Comment:
Is this correct?

Reply:
yes 
https://github.com/ONSdigital/ras-rm-performance-tests/pull/22/files#:~:text=GOOGLE_CLOUD_PROJECT%3A%20%22ras%2Drm%2Dperformance%2D20220908,%22ras%2Drm%2Dperformance%2D20220908%2Dlocust%22